creator-node/src/ipfsClient.js

```diff
@@ -0,0 +1,25 @@
+const ipfsClient = require('ipfs-http-client')
```

Comment:
i thought we already made these changes? does this need a rebase?

Reply:
ah yes. I made this PR before I merged my rehydrate ipfs PR, so I just cherry picked this file :). will rebase ~